### PR TITLE
Allow user to suppress global parameters info from help display

### PR DESCRIPTION
**Pull request by thrijith**
Fixes https://github.com/wp-cli/ideas/issues/104

### CODE REVIEW

**Comment by thrijith** [comment]
@schlessera how to check which step is undefined? 

Failure seems unrelated to change:

https://travis-ci.org/github/wp-cli/wp-cli/jobs/708051503#L454-L460
https://travis-ci.org/github/wp-cli/wp-cli/jobs/708051503#L346-L348

**Comment by schlessera** [comment]
@thrijith It looks like there's an instance where the step syntax contains a double space, which is the likely cause of the failure.

File: features/help.feature [hunk 1]
```diff
@@ -53,6 +53,40 @@ Feature: Get help about WP-CLI commands
...
+    Then STDOUT should not contain:
+      """
+      GLOBAL PARAMETERS
+      """
+
+    And STDOUT should not contain:
+      """
+      --path
+      """
+    And STDOUT should not  contain:
```
Comment (schlessera) [inline features/help.feature:85]: This seems to be the reason for the failing test.

```suggestion
    And STDOUT should not contain:
```

**Comment by thrijith** [comment]
> @thrijith It looks like there's an instance where the step syntax contains a double space, which is the likely cause of the failure.

Yes, that's it 🙂

**Comment by thrijith** [comment]
@schlessera there is one more in `trunk` https://travis-ci.org/github/wp-cli/wp-cli/jobs/708222192#L505-L511, but that seems to be the same as you mentioned on [#cli](https://wordpress.slack.com/archives/C02RP4T41/p1594227413299200)

**Comment by schlessera** [comment]
I'll see if I can fix that one today.

**Comment by schlessera** [comment]
Created a Core trac bug report here: https://core.trac.wordpress.org/ticket/50670

**Review by schlessera — APPROVED** [review]

**Comment by schlessera** [comment]
I added a new issue for documenting this: https://github.com/wp-cli/wp-cli/issues/5444.

**Pull request merged**